airbyte-integrations/connectors/source-github/source_github/graphql.py

```diff
@@ -87,12 +93,116 @@ def get_query_reviews(owner, name, first, after, number=None):
     )
     reviews.nodes.commit.oid()
     user = reviews.nodes.author(__alias__="user").__as__(_schema_root.User)
-    user.__fields__(
-        id="node_id",
-        database_id="id",
-        login=True,
-        avatar_url="avatar_url",
-        url="html_url",
-        is_site_admin="site_admin",
-    )
+    select_user_fields(user)
     return str(op)
+
+
+class QueryReactions:
+
+    AVERAGE_REVIEWS = 5
+    AVERAGE_COMMENTS = 2
+    AVERAGE_REACTIONS = 2
```

Comment:
Unclear what these numbers mean and where they come from... code comment would be good.